<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>7Z工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../config/icon/ico.qrc">
    <normaloff>:/文件/解压包.png</normaloff>:/文件/解压包.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>解压合并区</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="file_path_line"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="select_files_pushButton">
          <property name="text">
           <string>选择需要解压并合并的数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unpack_merge_pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>解压并合并</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../config/icon/ico.qrc"/>
 </resources>
 <connections/>
</ui>
